clients/admin-ui/src/features/dataset/DatasetTable.tsx

```diff
@@ -41,14 +41,19 @@ const DatasetsTable = () => {
   if (!datasets) {
     return <div>Empty state</div>;
   }
+
   return (
     <Table size="sm" data-testid="dataset-table">
       <Thead>
         <Tr>
           <Th pl={1}>Name</Th>
           <Th pl={1}>Fides Key</Th>
           <Th pl={1}>Description</Th>
-          <Th pl={1}>Status</Th>
+          {features.plus ? (
+            <Th data-testid="dataset-table__status-table-header" pl={1}>
```

Comment:
Nit: we haven't been using a BEM-ish naming structure for test IDs so far, though I see it could be helpful if there was going to be a naming clash on the page. I've used [cy.within](https://docs.cypress.io/api/commands/within#Syntax) when I want to make sure some elements are in a certain component, but it's a bit verbose.